sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/TransportReceiver.cs

```diff
@@ -32,6 +32,11 @@ internal abstract class TransportReceiver
         /// </summary>
         public abstract DateTimeOffset SessionLockedUntil { get; protected set; }
 
+        /// <summary>
+        /// The prefetch count associated with the receiver.
+        /// </summary>
+        public abstract int PrefetchCount { get; set; }
```

Comment:
At first glance I was on the fence to expose it here, but then I realized even though the abstract class says it should not have protocol-specific stuff it seems there is already a lot of protocol-specific stuff passed while creating the types, so I exposed it here since that is the central place where links are managed.

Reply:
Yes, this is also on the public type so it makes sense it would be on this exposed on this type as well.